_posts/2017/2017-07-19-reunification.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Reunification"
+layout: "post"
+tags: ["news"]
+---
+
+Having an individual repo for each driver is causing a lot of administrative overhead so we have decided to merge all repos into a single one. This has some implications you may have to address before and after this happens so I am going to try summarize them here:
+
+1. If you are **installing napalm fully** via the `napalm` package, ie `pip install napalm`:
+    1. Before the merge make sure you pin the version to `<2`. You can easily do that by updating your `requirements.txt` with the string `napalm<2`. If you don't have a requirements file `pip install "napalm<2"` will do the trick. Doing this will ensure nothing breaks for you.
+    1. After the merge, you will have to remove the `<2` constrain and adapt a few import paths. Don't worry, a few import paths is everything that will change. We will provide instructions to make this transition smoother.
+1. If you are **installing napalm partially by installing individual drivers** via their respective packages, i.e. `napalm-eos`, `napalm-junos`, etc...:
```

Comment:
If users won't be able to do `import napalm_ios` or `import napalm_junos` we have to let them know.